packages/twenty-front/src/modules/support/components/SupportChatEffect.tsx

```diff
@@ -0,0 +1,7 @@
+import { useSupportChat } from '@/support/hooks/useSupportChat';
+
+export const SupportChatEffect = () => {
+  useSupportChat();
```

Comment:
we should rename this hook to make clear it's doing stuff (effects): useInstantiateSupportChat